setupTests.js

```diff
@@ -12,3 +12,56 @@
 
 // setup file
 import '@testing-library/jest-dom';
+
+const ERROR_PATTERNS_WE_SHOULD_FIX_BUT_ALLOW = [
+  'An update to %s inside a test was not wrapped in act',
+  'Unknown event handler property `%s`',
+  'Not implemented: navigation',
+  'React does not recognize the `%s` prop on a DOM element',
+  'Received NaN for the `%s` attribute',
+  'Can\'t perform a React state update on an unmounted component',
+  '%s contains an input of type %s with both value and defaultValue props',
+  '<%s /> is using incorrect casing',
+  'The tag <%s> is unrecognized in this browser',
+  'The above error occurred in the <TestHook> component'
+];
+
+const WARNING_PATTERNS_WE_SHOULD_FIX_BUT_ALLOW = [
+  '`waitForDomChange` has been deprecated.',
```

Comment:
This might be a relatively straightforward warning to get through and fix.

Reply:
probably, do you want to do it here? (i'm fine if you don't)

Reply:
I'll take another look and if I can solve it in an hour or two I'll include the fixes in this PR.